python/paddle/fluid/tests/unittests/test_layers.py

```diff
@@ -410,6 +410,15 @@ def test_maxout(self):
             self.assertIsNotNone(output)
         print(str(program))
 
+    def test_mean_iou(self):
+        program = Program()
+        with program_guard(program):
+            x = layers.data(name='x', shape=[16], dtype='float32')
+            y = layers.data(name='label', shape=[1], dtype='int64')
+            iou = layers.mean_iou(x, y)
+            self.assertIsNotNone(loss)
+        print(str(program))
```

Comment:
Please help to change line 404 to 

```
def test_crop(self):
```

Thanks!

Reply:
Done.